common/v2/features/DeFiZap/config.ts

```diff
@@ -131,3 +143,78 @@ export const ZAPS_CONFIG: IZapConfigObject = {
     positionDetails: () => BearishIndicator({ text: 'Bearish on ETH' })
   }
 };
+
+export const riskAndReward: RiskAndReward[] = [
+  {
+    text:
+      '**Smart Contract:** there is a chance that  smart contracts get hacked and you lose all your money',
+    icon: smartContractIcn
+  },
+  {
+    text: '**Investing:** put your ETH to work for you, potential to watch your money grow',
+    icon: inverstingIcn
+  },
+  {
+    text:
+      '**Collateralization:** if the crypto price swings you don’t get liquidated nor does the entire system collapse',
+    icon: collateralizationIcn
+  },
+  {
+    text:
+      '**Innvoation:** Take advantage of innovative decentralized tools avaiable to ETH holders',
+    icon: innovationIcn
+  },
+  {
+    text: '**Liquidity:** markets are more inefficient when they are more shallow',
+    icon: liquidityIcn
+  },
+  {
+    text: '**Diversification:** Expand your investment portfolio leveraging your ETH',
+    icon: diversificationIcn
+  }
+];
+
+export const accordionContent = [
```

Comment:
We should create the accordionContent dynamically by merging multiple lists of questions & answers (general defi & defizap questions && a zap's protocol-specific questions). For example, when looking at the questions for `unipooldai` ZapID, we don't really care about Compound questions, but we do want to see Uniswap questions & answers.  But we'll still want to see the general DeFi / DFZ questions.

Reply:
Yup you're right ! Gonna wait for the complete list before doing that